packages/x-charts/src/context/CartesianProvider/computeValue.ts

```diff
@@ -12,31 +18,48 @@ import { FormattedSeries } from '../SeriesContextProvider';
 import { MakeOptional } from '../../models/helpers';
 import { getAxisExtremum } from './getAxisExtremum';
 
-const DEFAULT_CATEGORY_GAP_RATIO = 0.2;
-const DEFAULT_BAR_GAP_RATIO = 0.1;
-
-export const computeValue = (
+export function computeValue(
```

Comment:
```sugggestion

export function computeValue(
```

Reply:
I propose to keep those constant definition outside of the computation since they are just constant that could be modified without impacting the logic